promotion/service.go

```diff
@@ -217,8 +218,17 @@ func InitService(
 		}
 	}
 
-	var geminiClient gemini.Client
+	var (
+		geminiClient gemini.Client
+		geminiConf   gemini.Conf
+	)
 	if os.Getenv("GEMINI_ENABLED") == "true" {
+		geminiConf = gemini.Conf{
+			ClientID: os.Getenv("GEMINI_CLIENT_ID"),
```

Comment:
do you want to hard fail if these aren't present?

Reply:
https://github.com/brave-intl/bat-go/pull/783/commits/c825cc7b815f7b7b7ea6faa013dbffdf5056be0f <- resolved in here, don't want the service to not initialize, just the actual redeem and transfer to fail if not configured.